src/bank.rs

```diff
@@ -224,10 +235,15 @@ impl Bank {
         {
             let option = bals.get_mut(&tx.from);
             if option.is_none() {
-                if let Instruction::NewVote(_) = &tx.instruction {
-                    inc_new_counter_info!("bank-appy_debits-vote_account_not_found", 1);
+                // TODO(anatoly): this is gnarly because the counters are static atomics
```

Comment:
nit:  Please no `TODO($(whoami))` in the code.  "TODO" is sufficient, `git blame` will find you if needed.